github/ci/prow/files/jobs/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -548,7 +548,7 @@ presubmits:
     spec:
       nodeSelector:
         model: r430
```

Comment:
should the model change as well? since its not r430 anymore

Reply:
Yes, good catch. Thanks.

Reply:
Fixed.